washAOD/python/multicrab_iDMAnalysis.py

```diff
@@ -94,7 +94,9 @@ def main():
         config.General.transferLogs = True
 
         config.JobType.pluginName = 'Analysis'
-        config.JobType.psetName = 'iDMAnalyzer_cfg.py'
+        #config.JobType.psetName = 'iDMAnalyzer_cfg.py'
+        config.JobType.psetName = 'MetTrigSelfEffiForMuTrack_cfg.py'
```

Comment:
I think this will change the default multicrab_iDMAnalysis.py which should call the iDMAnalyzer_cfg.py. Can you double check this?